docs/README.md

```diff
@@ -1,12 +1,13 @@
+[Français](./fr)
```

Comment:
Instead of just linking to the French readme at the top of the main landing readme, do you think it might be better to have an english and a french section within the same document leading to their respective sections à la [nrcan_api](https://github.com/cds-snc/nrcan_api#lapi-de-l%C3%89nerguide-de-rncan)

Reply:
I think that's a good idea for the main README so both langs are in the forefront. I'd keep all the other docs separate at this point.